typing/typetexp.ml

```diff
@@ -461,7 +460,9 @@ and transl_type_aux env policy styp =
       unify_var env (newvar()) ty';
       ctyp (Ttyp_poly (vars, cty)) ty'
   | Ptyp_package (p, l) ->
-      let l, mty = create_package_mty true styp.ptyp_loc env (p, l) in
+      let loc = styp.ptyp_loc in
```

Comment:
If you are introducing this abbreviation, you may want to use below in line 471 as well.